_posts/2017-09-14-survey-results.md

```diff
@@ -0,0 +1,218 @@
+	* [Code changes](#code-changes)
+* [Conclusions](#conclusions)
+
+
+## About our users
+
+More than three quarters of our respondents are developers. 
+
+![Pie chart: Do you mainly see yourself as a... Developer 72%; QA tester 5%; Accessibility tester 6%; UX specialist 6%; Other 11%](asset/image/2017-survey/1-job-type.png)
+
+Over three quarters of our respondents use Pa11y for work. 
+
+20% of respondents use Pa11y only for personal projects, and 10% for both work and personal. There's no particular pattern in the way our respondents use Pa11y and the way they describe their job roles. 
+ 
+![Bar chart: What do you use Pa11y for?](asset/image/2017-survey/1-use-case.png)
+
+
+## Pa11y CLI
+
+The Pa11y command line tool is the “core” tool. It’s a simple command line interface that loads a URL and an optional configuration file, then runs HTML Codesniffer to generate a report on any accessibility errors it finds.  
+
+Two thirds of respondents have used the Pa11y CLI. 
+
+![Pie chart: Have you ever used the Pa11y command line tool? Used 67%; Not used 33%](asset/image/2017-survey/2-pa11y-cli.png)
+
+Of those who _haven’t_ used it, one third of respondents say that it’s too difficult to set up, and one third have simply never heard of it. The rest just don't have a need for it. 
+
+![Bar chart: Is there a reason that you've never used the Pa11y command line tool?](asset/image/2017-survey/2-not-used.png)
+
+Of those who _have_ used Pa11y CLI, we asked how easy/hard it is to set up. None of these respondents found it very difficult, but otherwise there is an even spread across the easy-to-hard spectrum. Most of the respondents who have used Pa11y CLI are developers; we’d expect this for a command-line tool. Greater familiarity with the command line amongst this group will also skew the results towards the “easier” end of the spectrum. 
+
+![Bar chart, scale of 1 (easier) to 5 (harder): How easy did you find it to set up the Pa11y CLI? 25% each for 1 to 4](asset/image/2017-survey/2-setup.png)
+
+Once set up, most people find Pa11y CLI easy to use. Those that found it more difficult tend to be doing more complex things with it, and overall people find it more difficult to integrate Pa11y CLI into their existing workflow. 
+
+![Bar chart, scale of 1 (easier) to 5 (harder): How easy did you find it to use the Pa11y CLI? 30% for 1; 25% for 2; 15% for 3; 25% for 4](asset/image/2017-survey/2-use.png)
+![Bar chart, scale of 1 (easier) to 5 (harder): How easy did you find it to integrate the Pa11y CLI with your workflow? 50% for 1; almost 15% for 2 and 3; 25% for 4](asset/image/2017-survey/2-integrate.png)
+
+When we asked “what would make Pa11y CLI easier for you to integrate with your workflow?”, some of our respondents who found Pa11y CLI a little harder to use and to integrate said the following:
+
+* Better examples for different situations -- integration with node or non-node apps.
```

Comment:
If these are direct quotes from the survey results, should we use `<q>`?

Reply:
yeah, i was just being lazy, really (hate typing html in markdown :D)